styles.css

```diff
@@ -1 +1,76 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+/*dit is de mobiele grid*/
+.container {
+    background-color: hsl(0, 30%, 95%);
+    display: grid;
+    grid-gap: 16px;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto 50px 50px 50px 50px 50px;
+    grid-template-areas:
+            "item-a"
+            "item-b"
+            "item-c"
+            "item-d"
+            "item-e"
+            " . "
+            "item-f";
+}
+
+.items {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+}
+
+#item-a {
+    background-color: hsl(350,75%,64%);
+    grid-area: item-a;
+    border: 2px solid darkred;
+}
+
+#item-b {
+    background-color: hsl(350,75%,64%);
+    grid-area: item-b;
+    border: 2px solid darkred;
+}
+
+#item-c {
+    background-color: hsl(350,75%,64%);
+    grid-area: item-c;
+    border: 2px solid darkred;
+}
+
+#item-d {
+    background-color: hsl(350,75%,64%);
+    grid-area: item-d;
+    border: 2px solid darkred;
+}
+
+#item-e {
+    background-color: hsl(350,75%,64%);
+    grid-area: item-e;
+    border: 2px solid darkred;
+}
+
+#item-f {
+    background-color: hsl(350,75%,64%);
+    grid-area: item-f;
+    border: 2px solid darkred;
+}
```

Comment:
Ik zie dat je bij elk item element dezelfde background-color en border gebruikt. Het is beter als je dit onder .items plaats. Het geldt dan voor alle items en het scheelt je hoop typen.